<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recognition</class>
 <widget class="QWidget" name="Recognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="enableAnimeRecognition">
        <property name="text">
         <string>Enable Anime Recognition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableMasato">
        <property name="text">
         <string>Enable Masato Anime Detection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableDiscord">
        <property name="text">
         <string>Enable Discord Rich Presence</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="updateDelay"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Update Delay:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Notification Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="notifyDetected">
        <property name="text">
         <string>Show a Notification When an Anime is Detected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="notifyUpdated">
        <property name="text">
         <string>Show a Notification When Shinjiru Updates an Anime's Progress</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
